src/main/resources/io/jenkins/plugins/designlibrary/AppBar/index.properties

```diff
@@ -1,2 +1,4 @@
 title=Page title
 description=App bars provide the page heading as well as important actions about the current page. It must be included before the <code>l:main-panel</code>.
+bottomAppBar.title=Bottom app bar
+bottomAppBar.description=Bottom app bars are great for forms, they provide important actions for the current page, such as Save and Apply. Bottom app bars stick to the bottom of the screen so that they're always present.
```

Comment:
```suggestion
bottomAppBar.description=Bottom app bars are great for forms, they provide important actions for the current page, such as "Save" and "Apply". Bottom app bars stick to the bottom of the screen so that they're always present.
```